lib/writeTempConfig.js

```diff
@@ -18,16 +19,24 @@ process.on('exit', function () {
   }
 });
 
-module.exports = function (config, esPath) {
+module.exports = function (config, logLevel, esPath) {
   return mkdirTemp({prefix: 'libesvm-'}).then(function createConfig(dir) {
-    var configFileDestination = path.join(dir, 'elasticsearch.json');
-
-    return Promise.all([
-      writeFile(configFileDestination, JSON.stringify(config), 'utf8'),
-      copy(join(esPath, 'config'), dir)
-    ])
+    return Promise.attempt(function () {
```

Comment:
I have a tendency to use attempt because I normally use it like so

```js
import { attempt } from 'bluebird';
// ...
  await attempt(...)
// ...
```